synapse/rest/client/v2_alpha/account.py

```diff
@@ -529,7 +529,7 @@ def on_POST(self, request):
 
 
 class ThreepidUnbindRestServlet(RestServlet):
-    PATTERNS = client_patterns("/account/3pid/unbind$")
+    PATTERNS = client_patterns("/account/3pid/unbind$", unstable=True)
```

Comment:
in that case we should remove it from r0 as well?

Reply:
Yes you're right, I have made it so.